pkg/cmd/generate-test-suites/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"sort"
+	"strings"
+)
+
+func main() {
+	// First list all tests.
+	buf, err := exec.Command("bazel", "query", "kind(go_test, //pkg/...)", "--output=label").Output()
+	if err != nil {
+		log.Printf("Could not query Bazel tests: got error %v", err)
+		os.Exit(1)
+	}
+	labels := strings.Split(string(buf[:]), "\n")
+	sort.Slice(labels, func(i, j int) bool { return labels[i] < labels[j] })
+
+	// Write the output to stdout
+	fmt.Println("# GENERATED FILE DO NOT EDIT")
```

Comment:
[nit] It can come in handy to specify how this is generated, "Code generated by generate-test-suites, DO NOT EDIT."